manifests/base/role.yaml

```diff
@@ -122,3 +122,27 @@ rules:
       - console.openshift.io
     resources:
       - odhquickstarts
+  - apiGroups:
+      - template.openshift.io
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
```

Comment:
FWIW, you can use `*`

```
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  namespace: default
  name: example.com-superuser # DO NOT USE THIS ROLE, IT IS JUST AN EXAMPLE
rules:
- apiGroups: ["example.com"]
  resources: ["*"]
  verbs: ["*"]
```
Source: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources

Reply:
Yeah, you are right, I think it's better to just use the wildcard rather than listing all the verbs.

Reply:
Since the goal is to do everything -- makes sense to use wildcard imo.